samples/common/mcumgr_bt_ota_dfu/Kconfig

```diff
@@ -19,7 +19,12 @@ config NCS_SAMPLE_MCUMGR_BT_OTA_DFU
 	imply FLASH_MAP
 	imply FLASH
 	imply MCUMGR_GRP_IMG
+	imply MCUMGR_GRP_IMG_REJECT_DIRECT_XIP_MISMATCHED_SLOT if \
+		NCS_SAMPLE_MCUMGR_BT_OTA_DFU_MCUBOOT_DIRECT_XIP_SUPPORT
+	imply MCUMGR_GRP_IMG_DIRECT_UPLOAD if \
+		NCS_SAMPLE_MCUMGR_BT_OTA_DFU_MCUBOOT_DIRECT_XIP_SUPPORT
 	imply MCUMGR_GRP_OS
+	imply MCUMGR_GRP_OS_BOOTLOADER_INFO
```

Comment:
Comment from @kapi-no from previous PR:

@philips77, can we have the Bootloader info command available in all MCUboot bootloader modes?

Reply:
I don't think XIP is possible for nRF53 at all.